chat-components/src/common/Constants.ts

```diff
@@ -152,7 +152,7 @@ export const AriaLabels = class {
     public static readonly TurnSoundOn = "Turn sound on";
     public static readonly Minimize = "Minimize";
     public static readonly EmailChatTranscriptPane = "Email Chat Transcript Pane";
-    public static readonly InputValidationPaneInput = "Email this chat transcript. This will be sent after your chat ends. Email address text area";
+    public static readonly InputValidationPaneInput = "Please provide e-mail address to send transcript to. The transcript will be sent after the chat ends. Email address text area";
```

Comment:
Does the transcript really send after the chat ends?

Reply:
yup